skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala

```diff
@@ -11,37 +11,39 @@ import scala.concurrent.duration.{FiniteDuration, _}
   * Check [[https://kafka.apache.org/documentation/#newconsumerconfigs]]
   */
 final case class ConsumerConfig(
-  common: CommonConfig                = CommonConfig.Default,
-  groupId: Option[String]             = None,
-  maxPollRecords: Int                 = 500,
-  maxPollInterval: FiniteDuration     = 5.minutes,
-  sessionTimeout: FiniteDuration      = 10.seconds,
-  heartbeatInterval: FiniteDuration   = 3.seconds,
-  autoCommit: Boolean                 = true,
-  autoCommitInterval: FiniteDuration  = 5.seconds,
-  partitionAssignmentStrategy: String = "org.apache.kafka.clients.consumer.RangeAssignor",
-  autoOffsetReset: AutoOffsetReset    = AutoOffsetReset.Latest,
-  defaultApiTimeout: FiniteDuration   = 1.minute,
-  fetchMinBytes: Int                  = 1,
-  fetchMaxBytes: Int                  = 52428800,
-  fetchMaxWait: FiniteDuration        = 500.millis,
-  maxPartitionFetchBytes: Int         = 1048576,
-  checkCrcs: Boolean                  = true,
-  interceptorClasses: List[String]    = Nil,
-  excludeInternalTopics: Boolean      = true,
-  isolationLevel: IsolationLevel      = IsolationLevel.ReadUncommitted
+  common: CommonConfig                       = CommonConfig.Default,
+  groupId: Option[String]                    = None,
+  maxPollRecords: Int                        = 500,
+  maxPollInterval: FiniteDuration            = 5.minutes,
+  sessionTimeout: FiniteDuration             = 10.seconds,
+  heartbeatInterval: FiniteDuration          = 3.seconds,
+  autoCommit: Boolean                        = true,
+  autoCommitInterval: Option[FiniteDuration] = Some(5.seconds),
```

Comment:
default is `none` please

Reply:
and we also need to add `apply` method with required `autoCommitInterval` for backward compatibility

Reply:
>and we also need to add apply method with required autoCommitInterval for backward compatibility

Reply:
done